static/app/views/organizationStats/usageTable.tsx

```diff
@@ -33,11 +33,12 @@ type Props = {
 
 export type TableStat = {
   accepted: number;
-  dropped: number;
   filtered: number;
+  invalid: number;
   project: Project;
   projectLink: string;
   projectSettingsLink: string;
+  rate_limited: number;
```

Comment:
we do not use snake_case in the frontend, but I decided to open an exception here so we could reuse this https://github.com/getsentry/sentry/blob/ab0d287991fbfed6328d90fb04250dee2b2c2a6f/static/app/views/organizationStats/usageStatsProjects.tsx#L61